### PR TITLE
add ensure load balancer e2e test

### PR DESCRIPTION
**Pull request by fseldow**
**What this PR does / why we need it**:
Add tests when updating service load balancer including:

1. assigning to specific IP when updating public service
2. updating internal IP when updating internal service
3. updating an internal service to a public service with assigned IP
4. no operation since no change in service when update

**Release note**:
```
none
```

### CODE REVIEW

File: tests/e2e/network/ensureloadbalancer.go [hunk 1]
```diff
@@ -0,0 +1,276 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```
Comment (feiskyer) [inline tests/e2e/network/ensureloadbalancer.go]: nit: 2019

**Comment by fseldow** [comment]
\reopen

**Pull request closed without merging**